charts/barman/templates/NOTES.txt

```diff
@@ -0,0 +1,23 @@
+The barman chart has been installed.
+
+Make sure your PostgreSQL server is properly setup to allow backups from barman.
+See: http://docs.pgbarman.org/release/2.10/#setup-of-a-new-server-in-barman
+
+
+Initial backup:
+
+  * Obtain a bash shell on your Barman server.
+
+    kubectl --namespace {{ .Release.Namespace }} exec -it {{ template "barman.fullname" . }}-0 /bin/bash
+
+  * Reset WAL log to perform first replication.
+
+    gosu barman barman switch-xlog --force --archive all
+
+  * Perform the initial backup
+
+    barman backup all
+
+  * Verify replication and bacups are working as expected.
+
+    barman check all
```

Comment:
Can you add a link to some further reading/how to restore things to the end of these notes?